### PR TITLE
Initial creation of fr_CA locale for Address

### PR DESCRIPTION
**Pull request by mbeaty-sf**
### What does this changes

Address French Canadian Address formats to fr_CA

### What was wrong

The addresses that were being created were not properly reflecting Canadian Provinces/Cities or French Canadian address formats

### How this fixes it

Created updates to allow for the formatting, suffixes, and prefixes of Canadian French addresses

Fixes #...

### CODE REVIEW

**Review by fcurella — CHANGES_REQUESTED** [review]
can you add tests for the new provider?

File: faker/providers/address/fr_CA/__init__.py [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+
+    def street_prefix(self) -> str:
+        """
+        :example: 'rue'
+        """
+        return self.random_element(self.street_prefixes)
+
+    def city_prefix(self) -> str:
+        """
+        :example: 'rue'
```
Comment (fcurella) [inline faker/providers/address/fr_CA/__init__.py:345]: is "rue" a city prefix?

**Review by fcurella — CHANGES_REQUESTED** [review]
Could you run `make lint` and fix the issues?

**Comment by fjudith** [comment]
@mbeaty-sf  would be great if you can get this merge

**Pull request closed without merging**